srcalc/src/main/java/gov/va/med/srcalc/domain/variable/AbstractVariable.java

```diff
@@ -174,4 +174,29 @@ public String toString()
     {
         return getDisplayName();
     }
+	
+	/**
+	 * Keys are unique, so two {@link AbstractVariable} are equal if their keys are equal.
```

Comment:
This is informative, but is not phrased as a method contract. Should be something like: "Indicates whether another AbstractVariable is equal to this one. Considers two variables equal if and only if ..."

Make sure to warn your users that this implementation assumes that Variable.getKey() is truly unique.